src/util/RegistrationFormFactory.js

```diff
@@ -113,16 +113,35 @@ const checkSubSchemas = function(fieldName, model, subSchemas, showError) {
     } else {
       ele.children('p').addClass('default-schema');
     }
+
+    // clear aria role and live-region for re-validation
+    ele.children('p')
+      .removeAttr('role')
+      .removeAttr('aria-live');
```

Comment:
Possibly worth seeing if this can live inside the `// validate` block but I think it's fine here too.

Reply:
true. i think it's safer outside so we know it's back to a "default" state before applying the role and aria-live attributes again in the failed condition